src/Rel_PT_Emission_Profile.jl

```diff
@@ -59,15 +68,29 @@ R_isco = r_isco(a_star, M)
 L_edd = 3e4*L_☼*(M/M_☼)
 Mdot = -L_edd/(c^2*η)
 mdot = 0.1*Mdot
+g = 500
 
 r_vals = LinRange(R_isco, 10*R_isco, 10000)
 
 f_r = f.(r_vals, R_isco, a_star, M)
 D_vals = diss.(r_vals, f_r)
 T_vals = temp.(D_vals)
+
+T_max = maximum(T_vals)
+r_max = r_vals[argmax(T_vals)]
+f_max = f(r_max, R_isco, a_star, M)
+
+#T_max = 10e5
+
+T_obs_vals = temp_obs.(r_vals, r_max, g, T_max, f_r, f_max)
     
 dissplot = plot(r_vals, D_vals, xlabel = "r(m)", ylabel = "Dissipation (W/m²)")
 tempplot = plot(r_vals, T_vals, xlabel = "r(m)", ylabel = "Temperature (K)")
+tempobsplot = plot(r_vals, T_obs_vals, xlabel = "r(m)", ylabel = "Temperature (K)")
 
 display(dissplot)
-display(tempplot)
\ No newline at end of file
+display(tempplot)
+display(tempobsplot)
+
+
+plot(tempplot, tempobsplot)
```

Comment:
If I recall correctly, `display` and `plot` do the same thing in this context? Is both needed?

Reply:
The second plot was there to display them on the same image, they're a bit squashed together on that one though, as we didn't get around to resizing them yet. I remember having some problems with display doing more than one on the same picture, although you might be right to get rid of the first display to make things a bit less messy.